build_resolvers/lib/src/analysis_driver.dart

```diff
@@ -33,17 +42,32 @@ AnalysisDriver analysisDriver(BuildAssetUriResolver buildAssetUriResolver,
 
   var logger = PerformanceLog(null);
   var scheduler = AnalysisDriverScheduler(logger);
+  var packages = Packages({
```

Comment:
[optional] This is taking up a really big chunk of the method - consider pulling it out to a utility method